main.cpp

```diff
@@ -78,43 +100,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+void buildHouse(int squareFeet = 0)
+{
+    ignoreUnused(squareFeet);
+}
 
 /*
  2)
- */
+ */
+int caloriesBurned(int hoursSlept)
+{
+    ignoreUnused(hoursSlept);
+    return {};
+}
 
 /*
  3)
- */
+ */
+void scooped()
+{//an empty function for scooping
+}

```

Comment:
Please provide some function arguments.
use `float` for one of them, because you haven't used any floats as function arguments yet.
Some of the arguments need to have default values
- [ ] revise